nirc_ehr/resources/queries/study/necropsyStatus.sql

```diff
@@ -0,0 +1,5 @@
+SELECT
+    demo.Id,
+    CASE WHEN d.QCState.Label = 'Request: Pending' OR d.QCState.Label = 'Review Required' THEN 'Necropsy Pending' ELSE demo.calculated_status END AS calculated_status
```

Comment:
Can you just check here if there's a necropsy pending instead of changing the calculated_status of the animal in the deaths trigger.